Sim/SimG4Fast/src/lib/FastSimActions.cpp

```diff
@@ -1,10 +1,17 @@
 #include "SimG4Fast/FastSimActions.h"
+
+// FCCSW
 #include "SimG4Fast/SaveParticlesTrackingAction.h"
+#include "SimG4Fast/InitializeModelsRunAction.h"
 
-FastSimActions::FastSimActions(): G4VUserActionInitialization() {}
+namespace sim{
+FastSimActions::FastSimActions(const std::string& aSmearingToolName): G4VUserActionInitialization(),
```

Comment:
Can you reformat the member initialiser list as indicated here?:
https://github.com/zaborowska/FCCSW/blob/restructure/doc/CppCodingStyleGuidelines.md#code-formatting